### PR TITLE
Add support for 1.17

### PR DESCRIPTION
**Pull request by Archengius**
Fix crash with CLI parsing version as number

### CODE REVIEW

**Review by rom1504 — CHANGES_REQUESTED** [review]
Why do we need to do this condensing here instead of in pchunk ?

**Comment by Archengius** [comment]
Because otherwise we would need to completely change dumped chunk format and create 32 lighting files for each chunk. I don't think it's reasonable, and to end up even with 2 files you still need to do the condensing on the dumper side.

**Comment by rom1504** [comment]
why not just store the packet fields as they are ? it's all serialized as json anyway https://github.com/PrismarineJS/minecraft-chunk-dumper/blob/2a0b920298e492444c89383a47afefb7a42fbc3e/index.js#L256

the objective of this package is to store the information in a format as raw as possible, so they can serve as test files in pchunk and other packages

if we do some additional preprocessing, then it will need to be duplicated in all user packages, better avoid it

**Comment by rom1504** [comment]
if the format changed, it changed, and pchunk/mineflayer will have to adapt anyway
if you think it should not have changed, that's something to fix in protocol.json

**Comment by Archengius** [comment]
Removed condensing, hopefully this is an acceptable storage format for lighting

File: index.js [hunk 1]
```diff
@@ -10,11 +10,25 @@ const util = require('util')
 const mc = require('minecraft-protocol')
 const debug = require('debug')('chunk-dumper')
 
+function flattenMaskArray (sectionBitMask) {
```
Comment (rom1504) [inline index.js]: this is not needed here and introduces boilerplate everywhere if used
prismarine-chunk needs to handle all transformations

**Review by rom1504 — APPROVED** [review]

**Pull request merged**